python/pyarrow/_csv.pyx

```diff
@@ -762,3 +764,53 @@ def open_csv(input_file, read_options=None, parse_options=None,
                  move(c_convert_options),
                  maybe_unbox_memory_pool(memory_pool))
     return reader
+
+
+def write_csv(output_file, data, include_header=True,
```

Comment:
In the parquet and feather api, it's first the data and then the file name

Reply:
reversed.  thanks.